model/Security/Classes/VulnAssessmentRelationship.md

```diff
@@ -0,0 +1,25 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# VulnAssessmentRelationship
+
+## Summary
+
+Asbtract ancestor class for all vulnerability assessments
+
+## Description
+
+VulnAssessmentRelationship is the ancestor class common to all vulnerability
+assessment relationships. It factors out the common properties shared by them.
+
+## Metadata
+
+- name: VulnAssessmentRelationship
+- SubclassOf: /Core/Relationship
+- Instantiability: Abstract
+
+## Properties
+
+- assessedElements
```

Comment:
`assessedElements` property should be change back to `assessedElement` as in a recent Tech meeting we were told all properties name must be singular.  I don't like it either but that something to discuss with SPDX Core.

Reply:
OK, its all singular now :+1: